package com.java.gaurang.exception;

public class exceptiondriver {
	public static void main(String[] args) {
		System.out.println(trycatch.divideNo(5, 2));
		
		System.out.println(trycatch.divideNo(4, 2));
		
		System.out.println(trycatch.divideNo(5, 0));
		
		System.out.println(trycatch.divideNo(8, 2));
		
	}

}
